homework/tests/homework.spec.js

```diff
@@ -0,0 +1,230 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+const data = [
+  'Prototype',
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9'
+]
+
+
+data.forEach(version => {
+  test.describe(version + ': Add', () => {
+    test('Adding 2 and 3 results in 5', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
```

Comment:
URL is repeated in every single test.
In case of url change -> there will be plenty of places in the test file to change it. Please declare a variable for this.